### PR TITLE
Adding loaded upgrade for ibmcloud on prow

### PR DESCRIPTION
**Pull request by shahsahil264**

### CODE REVIEW

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ibmcloud-4.16-nightly-x86-loaded-upgrade-from-4.15-loaded-upgrade-415to416-24nodes

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ibmcloud-4.16-nightly-x86-loaded-upgrade-from-4.15-loaded-upgrade-415to416-24nodes

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ibmcloud-4.16-nightly-x86-loaded-upgrade-from-4.15-loaded-upgrade-415to416-24nodes

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ibmcloud-4.16-nightly-x86-loaded-upgrade-from-4.15-loaded-upgrade-415to416-24nodes

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ibmcloud-4.16-nightly-x86-loaded-upgrade-from-4.15-loaded-upgrade-415to416-24nodes

**Comment by shahsahil264** [comment]
@paigerube14 can you PTAL and approve?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__ibmcloud-4.16-nightly-x86-loaded-upgrade-from-4.15.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      product: ocp
+      stream: nightly
+      version: "4.16"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: loaded-upgrade-415to416-24nodes
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__ibmcloud-4.16-nightly-x86-loaded-upgrade-from-4.15.yaml:31]:
```suggestion
- always_run: false
  as: loaded-upgrade-415to416-24nodes
```

**Review by qiliRedHat — CHANGES_REQUESTED** [review]
We use 'always_run: false' for the jobs that we don't want to run in cron.

**Comment by shahsahil264** [comment]
LGTM

**Comment by shahsahil264** [comment]
@qiliRedHat Can you PTAL and add the lgtm tag?

**Comment by qiliRedHat** [comment]
@shahsahil264 I've added /lgtm and /approve, you can see them in the right top corner 'Labels'. What is needed in this pr may be /pj-rehearse ack. Let me try.

**Pull request merged**